main.py

```diff
@@ -29,5 +29,9 @@
         from scoreboard.tests import data
         models.db.create_all()
         data.create_all()
+    elif 'runtests' in sys.argv:
+        from scoreboard.tests.utils_test import test_all
```

Comment:
This is just a temporary measure, I agree that we should establish a centralized testing system